### ISSUE TITLE
[Adaptive Cards-InputFormWithRTL.json]: Proper validation error message is not set for the 'Email' and 'Phone' edit field on after filling invalid data present on the 'InputFormWithRTL.json' card.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9230]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0

**Repro steps:**
1. Open the Adaptive Cards Visualizer application.
2. Navigate to the 'InputFormWithRTL.json' control present on left navigation using tab key and activate it using enter key.
3. 'InputFormWithRTL.json' adaptive card will appear on right sight of the pane.
4. Navigate to the 'Email' and 'Phone' edit field using tab key and enter any invalid value on the field.
5. Verify that, proper validation error message is set for the 'Email' and 'Phone' edit field on after filling invalid data present on the 'InputFormWithRTL.json' card or not.

**Actual Result:**
Proper validation error message is not set for the 'Email' and 'Phone' edit field on after filling invalid data present on the 'InputFormWithRTL.json' card. On after activating the 'Submit' button no error message displayed.

**Note:**
1. Same issue replicates for the 'Email' and 'Phone' edit fields present on the 'InputsWithValidation.json' card.

**Expected Result:**
A proper validation error message should be displayed and set for the 'Email' and 'Phone' edit field on after filling invalid data present on the 'InputFormWithRTL.json' card. Additionally, the error message should have a sufficient contrast ratio with their adjacent background color and the screen reader should properly announce the information regarding it.

**User Impact:**
Users who rely on screen readers or other assistive technologies will be impacted if proper validation error message is not set for the 'Email' and 'Phone' edit field on after filling invalid data present on the 'InputFormWithRTL.json' card due to which users are not informed of invalid data entry and might get confusion, and they may unknowingly submit incorrect information.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/error-identification.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/19bcf414-6038-45b6-95d7-f1ef5532b1b3)

https://github.com/user-attachments/assets/379d2ca8-0ed8-4ee8-815e-8fc20da7bd3c

### Card JSON

```JSON
N/A
```

**Comment by vagpt** [comment]
Since the product is getting deprecated, hence closing this issue.
